robinhood.spec.in

```diff
@@ -159,7 +157,7 @@ Annex tools for robinhood.
 %setup -q -n @PACKAGE@-%{version}
 
 %build
-./configure @ac_configure_args@ %{?configure_flags:configure_flags} \
```

Comment:
Can't confirm.

```
$ ./configure --without-lustre
[...]
$ make rpm
[...]
rpmbuild --without lustre  --with mysql   --without lhsm  --without shook  --without backup --define="_topdir `pwd`/rpms" --define="_prefix /usr" \
	--define="dbversion `mysql_config --version | cut -d "." -f 1-2`" \
        --define="_sysconfdir /etc" \
	--define="_prefix /usr" -tb robinhood-3.0.tar.gz
[...]
$ echo $?
0
$ for i in rpms/RPMS/x86_64/robinhood-*; do sudo yum deplist $i; done | grep lustre | wc -l
0
```

Anything that points to lustre support being included on your system?